src/main/kotlin/no/nav/syfo/metrics/MetricRegistry.kt

```diff
@@ -63,3 +63,9 @@ val VIRKSOMHETSYKMELDING: Counter = Counter.build()
     .name("virksomhetsykmelding")
     .help("Antall virksomhet-sykmeldinger")
     .register()
+
```

Comment:
Kan vi ha en metrikk for når vi avviser pga virus også?

Reply:
Det kan vi :)